.changes/add-operation-object.md

```diff
@@ -0,0 +1,11 @@
+---
+"@effection/fetch": "patch"
```

Comment:
Only the test case changed, right?

```suggestion
```

Reply:
Yep. Do I need remove `patch` for other packages, where I changed only devDeps, too?

Reply:
I think for the typescript, it is ok to leave the patch in since it could _theoretically_ cause the package to be compiled differently thereby changing what JS gets shipped to NPM